wasm2c/examples/rot13/Makefile

```diff
@@ -1,9 +1,17 @@
 # Use implicit rules for compiling C files.
 CFLAGS=-I../..
-rot13: main.o rot13.o ../../wasm-rt-impl.o
 
-rot13.wasm: rot13.wat
+all: rot13
+
+clean:
+	rm -rf rot13 rot13.wasm rot13.c *.o
+
+rot13: main.o rot13.o ../../wasm-rt-impl.o ../../wasm-rt-os-unix.o -lm
```

Comment:
`../../wasm-rt-os-unix.o` doesn't (yet) exist on this branch.

Reply:
Oops thanks